Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -377,7 +382,7 @@ variable [CommSemiring R] [AddCommMonoid M] [Module R M]
 section SMul
 
 variable [Monoid S] [Monoid T] [DistribMulAction S R] [DistribMulAction T R]
-variable [SMulCommClass S R R] [SMulCommClass T R R]
+variable [SMulCommClass R S R] [SMulCommClass S R R] [SMulCommClass T R R] [SMulCommClass R T R]
```

Comment:
This looks like a regression to me; one implies the other so there should be no need for both

Reply:
Addressed in #10167